add_pressreview.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+
+import argparse
+import io
+import os
+import re
+import sys
+import unicodedata
+import urllib.error
+import urllib.request
+
+
+class PressReview:
+    """
+    Model for a press review entry.
+    """
+    _punct_re = re.compile(r'[\t !"#$%&\'()*\-/<=>?@\[\\\]^_`{|},.]+')
+
+    def __init__(self, date, publisher, title, url):
+        self.date = date
+        self.publisher = publisher
+        self.title = title
+        self.url = url
+        self.lang = 'de'
+
+    @property
+    def slug(self):
+        result = []
+        for word in self._punct_re.split(self.publisher.lower()):
+            word = unicodedata.normalize('NFKD', word)
+            if word:
+                result.append(word)
+        slug = '%s_%s' % (self.date, '-'.join(result))
```

Comment:
I think the existing slugs are wrong. Pelican's default for `FILENAME_METADATA` is:

```
FILENAME_METADATA ('(?P<date>\d{4}-\d{2}-\d{2}).*')
```
http://docs.getpelican.com/en/3.1.1/settings.html